server/src/services/wishlist.js

```diff
@@ -0,0 +1,20 @@
+const WishLists = require("../db/mongo/models/wishlist");
+const Series = require("../db/mongo/models/series");
+
+const searchWishlistByUsername = async (username) => {
+
+    const wishlist = await WishLists.findOne({ username: username}).exec();
```

Comment:
There are few ways to handle this:
1. You can populate series_ids and then it'll give you an array with all series from "series" collection, however as I've witnessed in `getUserWatchlist` it actually makes a query per id, similar to what you do here.
2. (Better option) After you get `series_ids`, query ONLY ONCE to get all series using $in. Way better performance wise and more elegant :)